inferno-core/CHANGELOG.md

```diff
@@ -5,6 +5,14 @@
 * Added asTensor0 function to inferno-ml
 * Added instructions for creating and compiling scripts locally with vscode to the README
 
+## 0.3.5 -- 2023-08-07
+* Created documentation for importing torchscript models
+
+## 0.3.4 -- 2023-08-02
+* Added stack function to inferno-ml
+* Added asTensor0 function to inferno-ml
+* Added instructions for creating and compiling scripts locally with vscode to the README
+
```

Comment:
These lines are already present above. Perhaps you intended to delete the above lines